CHANGELOG.md

```diff
@@ -12,10 +12,13 @@
 
 - Add configurable moment locales (INDIGO Sprint 210416) [!98] (https://github.com/TeskaLabs/asab-webui/pull/98)
 
+
 ### Refactoring
 
 - Renaming configuration option FAKE_USERINFO to MOCK_USERINFO and refactoring code accordingly (INDIGO Sprint 210406, [!91](https://github.com/TeskaLabs/asab-webui/pull/91))
 
 - Moving ASAB-Config module navigation under the `Maintenance` tab in the sidebar (INDIGO Sprint 210406, [!92](https://github.com/TeskaLabs/asab-webui/pull/92))
 
 - Refactor userinfo API call to obtain Roles and Resources for active tenant (INDIGO Sprint 210416, [!94](https://github.com/TeskaLabs/asab-webui/pull/94))
+
+- Refactor microservices list (INDIGO Sprint 210430, [!99](https://github.com/TeskaLabs/asab-webui/pull/99))
```

Comment:
Here should be written what was the actual refactorisation (the purpose of it, what it changed, etc.). This is too vague

Reply:
Got it!